zephyr/wrapper.c

```diff
@@ -107,11 +111,21 @@ static void *heap_alloc_aligned(struct k_heap *h, size_t min_align, size_t bytes
 {
 	k_spinlock_key_t key;
 	void *ret;
+#if CONFIG_SYS_HEAP_RUNTIME_STATS
+	struct sys_memory_stats stats;
+#endif
 
 	key = k_spin_lock(&h->lock);
 	ret = sys_heap_aligned_alloc(&h->heap, min_align, bytes);
 	k_spin_unlock(&h->lock, key);
 
+#if CONFIG_SYS_HEAP_RUNTIME_STATS
+	sys_heap_runtime_stats_get(&h->heap, &stats);
+#if CONFIG_IPC_MAJOR_4   // ipc3 may have issue when print logs
+	tr_info(&zephyr_tr, "heap allocatd: %x free: %x", stats.allocated_bytes, stats.free_bytes);
+#endif
```

Comment:
any idea what kind of issues ipc3 might? Also please use C style comments like this `/* ... */`

Reply:
once I added for ipc3, then seems no log output from terminal, I don't have details regarding this, since I only have one setup, and changed from ipc3 to ipc4, regarding the comments, I am changing.

Reply:
ack, this should work with any IPC version as it's coming from Zephyr.

Reply:
ok， maybe my test machine have something wrong at that time, I will remove ipc4 only.